ledfx_docker/Dockerfile

```
@@ -0,0 +1,43 @@
+FROM ghcr.io/astral-sh/uv:python3.12-bookworm AS build
+ENV UV_COMPILE_BYTECODE=1 UV_LINK_MODE=copy
+ENV UV_PYTHON_DOWNLOADS=0
+WORKDIR /app
+#RUN apt update && apt install -y cmake gcc portaudio19-dev && rm -rf /var/lib/apt/lists/*
+RUN --mount=type=cache,target=/root/.cache/uv \
+    --mount=type=bind,source=uv.lock,target=uv.lock \
+    --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
+    uv sync --frozen --no-install-project --no-dev
+ADD . /app
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)